CODEOWNERS

```diff
@@ -44,10 +42,10 @@
 /prow/jobs/kyma/components/istio-installer @tomasz-smelcerz-sap @werdes72 @pk85 @dariusztutaj @strekm @kyma-project/prow
 /prow/jobs/kyma/components/kyma-operator @tomasz-smelcerz-sap @werdes72 @pk85 @dariusztutaj @strekm @kyma-project/prow
 /prow/jobs/kyma/components/permission-controller @pk85 @dariusztutaj @strekm @tomasz-smelcerz-sap @werdes72 @kyma-project/prow
-/prow/jobs/kyma/components/telemetry-operator @pk85 @szwedm @tillknuesting @a-thaler @ralikio @wozniakjan @piotrmiskiewicz @kyma-project/prow
-/prow/jobs/kyma/components/directory-size-exporter @pk85 @szwedm @tillknuesting @a-thaler @ralikio @wozniakjan @piotrmiskiewicz @kyma-project/prow
+/prow/jobs/kyma/components/telemetry-operator @a-thaler @kyma-project/prow
```

Comment:
Should kyma-team/gopher be removed here?

Reply:
afaict yes, asked the whole @kyma-project/gopher team and no one knows what it is and looks like nobody from the gopher team ever contributed there
```
$ git log --pretty=format:%an prow/jobs/kyma/components/telemetry-operator/ | sort | uniq
Christoph Kleineweber
Dennis Geiselmann
Hisar Balik
Kamil Kasperski
Kyma Bot
Piotr Halama
Przemek Pokrywka
```

Reply:
@a-thaler it looks like you are the last owner from development teams for these 2 jobs. Are these jobs still needed?